compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -109,6 +109,10 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
            [](CompilationOptions &options, double global_p_error) {
              options.optimizerConfig.global_p_error = global_p_error;
            })
+      .def("set_allow_composition",
```

Comment:
maybe auto_composition instead of composition. How will it be called for composition of a set of function ?